src/core/src/FabricMaterialDefinition.cpp

```diff
@@ -26,6 +27,7 @@ FabricMaterialDefinition::FabricMaterialDefinition(
     _hasVertexColors = materialInfo.hasVertexColors;
     _hasBaseColorTexture = hasBaseColorTexture;
     _imageryLayerCount = imageryLayerCount;
+    _tilesetMaterialPath = tilesetMaterialPath;
```

Comment:
Super nit, but I'd probably do this and the above line as an initializer list. Any thoughts on what to do for consistency or other reasons?

Reply:
Fixed in https://github.com/CesiumGS/cesium-omniverse/pull/406/commits/67024a8887c3adff643a1d70c44c6ee98b51a498